data.json

```diff
@@ -179,6 +179,15 @@
     "dateAdded": "2018-10-23T10:17:39.000Z",
     "tags": ["clothing", "hacktoberfest", "expired"]
   },
+  {
+    "name": "MayaData",
+    "difficulty": "medium",
+    "description": "At the end of the month-long event, the individual who makes the most significant contributions will receive a brand new laptop.",
```

Comment:
It looks like you can also get other swag for contributing in general

Reply:
👍